src/main/java/com/hyperwallet/clientsdk/model/DocumentVerificationReason.java

```diff
@@ -0,0 +1,23 @@
+package com.hyperwallet.clientsdk.model;
+
+public enum DocumentVerificationReason {
```

Comment:
Could you refer HyperwalletUser? then we may not need all these enums with such details

Reply:
I think we need enums like this for description text?